meta-resin-common/recipes-connectivity/dnsmasq/files/dnsmasq.conf.systemd

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Wants=rce.service resin-net-config.service
+After=rce.service resin-net-config.service
```

Comment:
This will be needed when dnsmasq will bind to rce0 when we will be using networking namespacing for the containers